### PR TITLE
K6 0.31

### PR DESCRIPTION
**Pull request by ppcano**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/loadimpact/k6/releases/tag/v0.31.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/k6/index.d.ts [hunk 1]
```diff
@@ -116,14 +117,9 @@ export interface Checkers<VT> {
 // --------------------
 
 /**
- * Byte represented as number. Value range [0,256)
+ * Array of numbers [0, 256].
```
Comment (na--) [inline types/k6/index.d.ts:120]:
```suggestion
 * Array of numbers [0, 255].
```

**Review by na-- — APPROVED** [review]
LGTM besides the byte range

**Comment by ppcano** [comment]
Ready to merge

**Review by na-- — APPROVED** [review]

**Comment by ppcano** [comment]
Ready to merge

**Pull request merged**